packages/ui/src/views/repo/pull-request/compare/pull-request-compare-page.tsx

```diff
@@ -64,15 +62,15 @@ export interface PullRequestComparePageProps extends Partial<RoutingProps> {
   mergeability?: boolean
   onSelectCommit: (commit: CommitSelectorListItem) => void
   selectBranch: (branchTag: BranchSelectorListItem, type: BranchSelectorTab, sourceBranch: boolean) => void
-  targetBranch: BranchSelectorListItem
-  sourceBranch: BranchSelectorListItem
+  targetBranch: BranchSelectorListItem | null
```

Comment:
why `null`? why not make it optional?

```suggestion
  targetBranch?: BranchSelectorListItem
```